src/controllers/v1/profile.js

```diff
@@ -97,6 +97,27 @@ module.exports = class Mentees {
 		}
 	}
 
+	/**
+	 * Filter list
+	 * @method
+	 * @name filterList
+	 * @param {Object} req - request data.
+	 * @param {String} req.decodedToken.token - user token.
+	 * @returns {JSON} - filter list.
+	 */
+
+	async filterList(req) {
+		try {
+			const filterList = await menteesService.getFilterList(
+				req.query.entity_types ? req.query.entity_types : '',
+				req.decodedToken
+			)
+			return filterList
+		} catch (error) {
+			return error
```

Comment:
Do not return the error here. Console.log it and throw again. 

Otherwise, remove the try catch block altogether. A catch block that just throws an execption isn't doing anything.